extension/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "lmsx-extension-builder",
+  "version": "1.0.0",
+  "description": "Build and deploy for the LMSX extension",
+  "repository": "https://github.com/Ed-Fi-Alliance-OSS/LMS-Toolkit",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "atom-metaed": "3.0.1-dev.5"
+  },
+  "scripts": {
+    "prep:lnx": "cp -r ./EdFi.Ods.Extensions.LMSX ../../Ed-Fi-ODS-Implementation/Application/EdFi.Ods.Extensions.LMSX",
+    "prep:win": "xcopy .\\EdFi.Ods.Extensions.LMSX ..\\..\\Ed-Fi-ODS-Implementation\\Application\\EdFi.Ods.Extensions.LMSX /S /I",
+    "deploy": "node ./node_modules/@edfi/metaed-odsapi-deploy/dist/index.js --source ./node_modules/@edfi/ed-fi-model-3.3b --source ./EdFiLMSMetaEd --target ../../ --defaultPluginTechVersion 5.3.0",
+    "keep:lnx": "cp -r ../../Ed-Fi-ODS-Implementation/Application/EdFi.Ods.Extensions.LMSX/* EdFi.Ods.Extensions.LMSX",
+    "keep:win": "xcopy ..\\..\\Ed-Fi-ODS-Implementation\\Application\\EdFi.Ods.Extensions.LMSX\\* .\\EdFi.Ods.Extensions.LMSX /S"
```

Comment:
This file helps with building the the LMSX extension from the command line, without using Atom.